pom.xml

```diff
@@ -343,6 +332,14 @@
                 </execution>
               </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <configuration>
+                <systemPropertyVariables>
+                  <jacoco-agent.destfile>${jacoco.exec.file}</jacoco-agent.destfile>
+                </systemPropertyVariables>
+              </configuration>
+            </plugin>
```

Comment:
Effective `pom.xml` of  this module has this property:
```
<jacoco.exec.file>[DROOLS-WB-HOME]/drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/target/jacoco.exec</jacoco.exec.file>
```

However without this `jacoco-agent.destfile` the jacoco exec file is produced in `[DROOLS-WB-HOME]/drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/jacoco.exec`

Reply:
Thanks for confirmation, so it's definitely needed.